example/usps_api/config/config_parser.h

```diff
@@ -34,10 +38,14 @@ namespace usps_api_server {
       bool query;
```

Comment:
Member variable names should end with an _, https://google.github.io/styleguide/cppguide.html#Variable_Names.

Reply:
Done!